examples/Makefile

```diff
@@ -16,7 +16,7 @@
 
 TARGETS := ad9361-iiostream ad9371-iiostream adrv9009-iiostream dummy-iiostream iio-monitor
 
-CFLAGS = -Wall -DIIO_CHECK_RET
+CFLAGS = -Wall -Wextra -pedantic-errors -DIIO_CHECK_RET
```

Comment:
Shall `-pedantic-errors` (or just `-pedantic` be set when compiling libiio and iiod also?

Reply:
yes - I switched to `-pedantic`. Thanks